anima-jws-voucher.mkd

```diff
@@ -55,95 +55,101 @@ informative:
   RFC8949:
   RFC8792:
   RFC8812:
-  onpath:
+  ONPATH:
```

Comment:
Convention to usually use upper case or CapitalizedCamelCase for long references.